src/main/kotlin/org/veupathdb/lib/blast/blastn/BlastN.kt

```diff
@@ -10,245 +10,162 @@ import org.veupathdb.lib.blast.common.fields.*
 interface BlastN : BlastQueryWithLists {
 
```

Comment:
Stripped out everything but the CLI flag as the actual documentation will now live with the field implementation.